lib/ansible/modules/cloud/digital_ocean/digital_ocean_domain.py

```diff
@@ -108,7 +114,7 @@ def find(self):
         return False
 
     def add(self):
-        params = {'name': self.domain_name, 'ip_address': self.domain_ip}
+        params = {'name': self.domain_name, 'ip_address': self.domain_ip, 'ttl': self.domain_ttl}
         resp = self.post('domains/', data=params)
```

Comment:
I just realised that this is using a different endpoint from the DO API than I was expecting :man_facepalming: 

It would need to be changed to the `Domain Records` API. See `Domains` vs. `Domain Records` in the [DO API docs](https://developers.digitalocean.com/documentation/v2/#domain-records)

How should I proceed with this, should this module be changed to use those other (more configurable) endpoints, or is it a case of making a new module to more accurately reflect this separation in the API?